spec/support/rails_stub.rb

```diff
@@ -0,0 +1,17 @@
+module RailsStub
+  extend RSpec::Mocks::ExampleMethods
+
+  module_function
```

Comment:
TIL you can not pass arguments and the susequently defined methods become module functions!

Reply:
Hum, can't we do `module_function :my_method` as well (if that's what you mean)?

I think it works much like the `private` modifier.

Reply:
Right! I didnt't know it worked like private, thanks!